types/index.d.ts

```diff
@@ -0,0 +1,10 @@
+export * from './controllers';
+export * from './core';
+export * from './elements';
+export * from './core/interfaces';
+export * from './platform';
+export * from './plugins';
+export * from './scales';
+export * from './interfaces';
+
+export * from './helpers';
```

Comment:
Same as `helpers/index.d.ts`: looking at the code, it should reflect that the library actually has a single `export`. (May I ask, have you guys found issues with `export`ing separate items rather than having a single default export?)